scan/lib/scan/detect_values.rb

```diff
@@ -162,6 +162,7 @@ def self.detect_simulator(devices, requested_os_type, deployment_target_key, def
               filter_simulators(simulators, :equal, version).tap(&potential_emptiness_error).select(&selector)
             end
           ).tap do |array|
+            fail_if_device_not_found(device_string) if Scan.config[:ensure_devices_found] && array.empty?
             UI.error("Ignoring '#{device_string}', couldn’t find matching simulator") if array.empty?
```

Comment:
```suggestion
            if array.empty?
              UI.test_failure!("No '#{device_string}' found") if Scan.config[:ensure_devices_found]
              UI.error("Ignoring '#{device_string}', couldn’t find matching simulator")
            end
```

Reply:
Now I got it 😅😅
You did not, but I did 😉🙂

Could you please retry "Module up load tests" job ? I think it is a random failure.

Reply:
Never mind I merged master